cli/server.go

```diff
@@ -86,7 +87,7 @@ func server() *cobra.Command {
 		tlsKeyFile                       string
 		tlsMinVersion                    string
 		turnRelayAddress                 string
-		tunnel                           bool
+		shouldTunnel                     bool
```

Comment:
I've been hesitant to let these steer away from the defined names on the CLI. I think the visual consistency is helpful!